File: Python/baekjoon/단계별로 풀어보기/집합과 맵/14425_문자열 집합.py
# https://www.acmicpc.net/problem/14425

from sys import stdin

input = stdin.readline
a, b = map(int, input().split())
s = set()
for _ in range(a):
    s.add(input())
cnt = 0
for _ in range(b):
    if input() in s:
        cnt += 1
print(cnt)

File: C#/Baekjoon/집합과 맵/14425_문자열 집합.cs
//https://www.acmicpc.net/problem/14425

int[] n = Array.ConvertAll(Console.ReadLine().Split(), int.Parse);
int cnt = 0;
var S = new HashSet<string>();
while (n[0]-- > 0) S.Add(Console.ReadLine());
while (n[1]-- > 0) if (S.Contains(Console.ReadLine())) cnt++;
Console.Write(cnt);